cmd/minishift/cmd/root.go

```diff
@@ -52,10 +52,18 @@ import (
 
 const (
 	showLibmachineLogs    = "show-libmachine-logs"
+	profileCmd            = "profile"
 	profileFlag           = "profile"
```

Comment:
@praveenkumar I thought we agreed on `profileFlag           = "--profile"` . Any reason why we have not done that?

Reply:
The issue is we can not have `profileCmd` and `profileFlg` with same value i.e. `profile` , it looks odd.

Reply:
I do not think this is odd... writing the full flag in my opinion is, as this puts logic and an implementatiuon detail inside the flagname, while this should be generic logic.